controllers/admin_ctrl.js

```diff
@@ -115,6 +116,28 @@ async function stats(req, res) {
     }
   });
 }
+
+function addTimer(req,res, next){
```

Comment:
should delete old timers and you can move the whole function to questions_ctrl as it is related to the questions